src/net.cpp

```diff
@@ -2470,7 +2470,7 @@ void CNode::BeginMessage(const char* pszCommand) EXCLUSIVE_LOCK_FUNCTION(cs_vSen
     ENTER_CRITICAL_SECTION(cs_vSend);
     assert(ssSend.size() == 0);
     ssSend << CMessageHeader(Params().MessageStart(), pszCommand, 0);
-    LogPrint("net", "sending: %s ", SanitizeString(pszCommand));
+    LogPrint("net", "sending: %s\n", SanitizeString(pszCommand));
```

Comment:
I remember I saw at least one instance which was wrong but I can't recall which one as I have deleted the `debug.log` in the meantime. I have reverted for now and will look into this later.